sdk/keyvault/keyvault-keys/tests/list.test.ts

```diff
@@ -114,25 +114,24 @@ describe("Keys client - list keys in various ways", () => {
 
   it("list 0 versions of a non-existing key", async () => {
     let totalVersions = 0;
-    for await (let version of client.listKeyVersions(keyName)) {
+    for await (const version of client.listKeyVersions(keyName)) {
+      if (!version) break;
```

Comment:
This alters the previous behavior. In what cases do we have an empty version?

Reply:
there's no way we have an empty version! The issue here is that I can't have a variable I'm not using.